primary/src/helper.rs

```diff
@@ -86,6 +87,11 @@ impl Helper {
                     PrimaryMessage::CertificatesRequest(digests, origin) => {
                         let _ = self.process_certificates(digests, origin, false).await;
                     }
+                    // The CertificatesRangeRequest will collect certificate digests from range_start,
+                    // for at most max_rounds number of rounds. All digests will be returned in one response.
+                    PrimaryMessage::CertificatesRangeRequest {range_start, max_rounds, requestor: origin} => {
+                        let _ = self.handle_certificates_range_request(range_start, max_rounds, origin).await;
```

Comment:
I know you are following the executing pattern, but why is it ok to ignore the result of this function?

Reply:
Perhaps we want to log it in case it is a store error?